GHSA-4grx-2x9w-596c.analysis.json

```diff
@@ -0,0 +1 @@
+{"firstIssued": "2024-06-05T15:32:43Z","state": "under_investigation","detail": "No fixed upgrade is available!"}
```

Comment:
should we put these in somewhere centralized like a subfolder of `resources`?

Reply:
Yes, definitely!
I added the change.